File: backend/module-auth/src/main/kotlin/com/lighttalk/auth/dto/PhoneLoginRequest.kt
package com.lighttalk.auth.dto

import jakarta.validation.constraints.NotBlank

data class PhoneLoginRequest(
    @field:NotBlank(message = "전화번호는 필수입니다")
    val phoneNumber: String,

    @field:NotBlank(message = "비밀번호는 필수입니다")
    val password: String
)

File: frontend/lib/features/auth/data/models/phone_login_request.dart
class PhoneLoginRequest {
  final String phoneNumber;
  final String password;

  const PhoneLoginRequest({
    required this.phoneNumber,
    required this.password,
  });

  Map<String, dynamic> toJson() => {
        'phoneNumber': phoneNumber,
        'password': password,
      };
}
